### ISSUE TITLE
Does not show my serverlist

### ISSUE THREAD

**Issue by KaneSama2k3** [Pugmatt/BedrockConnect#343]
### What happened?

I have a pair of Minecraft Bedrock docker servers on the same machine, one with the port 19135 and the other with 19134 that I want to be able to connect to using the switch / PS4 on my local network so I was trying to use Bedrock connect for this.
I have a pi-hole as the network DNS and added local settings to override the featured servers list and point to the IP for my docker server.

I set up a BedrockConnect Server using a slight modification to the dockerfile in the project files

All of the featured servers are showing up, but clearly not being able to connect as they show no extra description or ping information.

On the PC, I can connect to the two servers I have created without issue

### Expected Behaviour?

Based on the "custom_servers=custom_servers.json", "featured_servers=false" arguments, it should only show either my servers or any featured servers I have not overwritten, unless I'm misunderstanding how it's meant to work

### Steps to reproduce.

Dockerfile:
FROM openjdk:11
RUN mkdir -p /docker/brc
ADD https://github.com/Pugmatt/BedrockConnect/releases/latest/download/BedrockConnect-1.0-SNAPSHOT.jar /docker/brc
WORKDIR /docker/brc
EXPOSE 19132/udp
COPY ./custom_servers.json /docker/brc/custom_servers.json
CMD ["java", "-Xms256M", "-Xmx256M", "-jar", "BedrockConnect-1.0-SNAPSHOT.jar", "nodb=true", "custom_servers=custom_servers.json", "featured_servers=false"]

Docker run:
docker run -d  -p 19132:19132

custom_servers.json
[
	{
		"name": "Shared server",
		"address": "ipaddress of docker server not instance",
		"port": 19134
	},
	{
		"name": "Server2 Server",
		"address": "ipaddress of docker server not instance",
		"port": 19135
	}
]

### Minecraft Bedrock Version

1.16

### Console

Nintendo Switch, PS4

**Comment by Pugmatt** [comment]
```
All of the featured servers are showing up, but clearly not being able to connect as they show no extra description or ping information.
```
The BedrockConnect serverlist does not display descriptions or ping, it can't do that since the form data is served by the BedrockConnect server. Are you talking about the menu under the "Servers" tab in the "Play" menu? If so, at that point you aren't on the BedrockConnect serverlist yet, you connect to the BedrockConnect serverlist after joining one of the featured servers. (Which the custom DNS server spoofs into instead connecting to the BedrockConnect serverlist) So it should be displaying the featured servers either way in that tab. 

If the featured servers in the "Servers" tab show no sign of connection, that would mean your game console is not seeing the BedrockConnect instance hosted locally on your network. You might have a firewall issue, or I would double-check the IP address you have in the DNS record for your DNS server.

**Comment by KaneSama2k3** [comment]
I see; I did wonder how it would spoof the entire server-list; is there a way to see connection logs on the BedrockConnect server so I can work out why I get the error 'cannot connect to world'? All I see in the docker log is  the below:

![image](https://user-images.githubusercontent.com/46218948/195039708-b01cb229-5bcb-4542-a138-fa1c5a5d67f6.png)

I'm confident there's not an IP address issue as all of the IP addresses are pointing to the docker host that is running both the BedrockConnect docker instance and the two bedrock Minecraft servers (running on adjacent ports).
If I connect to the IP address / port directly on Windows, it works fine, but not through featured servers.

**Comment by KaneSama2k3** [comment]
So I have a Nas running docker with all the instances on it.
I then have a pothole for my DNS anyway that I have added the a rule to point to the Nas
When I only had one server set up, I was able to connect to it using the featured servers without bedrock connect.
When I tried to add a second server I realised I'd have the port problem which is why I tried to set up bedrock connect instead and changed the ports on the Minecraft servers.
Are there any other ports I need to expose on the docker instance??
